websites/C/Character.AI/presence.ts

```diff
@@ -0,0 +1,58 @@
+const presence = new Presence({
+    clientId: "1061324603022114998",
+}),
+browsingTimestamp = Math.floor(Date.now() / 1000);
+
+presence.on("UpdateData", () => {
+const presenceData: PresenceData = {
+    largeImageKey: "logo",

```

Comment:
We're changing all largeImageKeys to imgur links
```suggestion
    largeImageKey: "https://i.imgur.com/eIpvMGf.png",

```